frontend/server/src/main/java/org/pytorch/serve/util/ConfigManager.java

```diff
@@ -68,6 +68,16 @@ public final class ConfigManager {
 
     // IPEX config option that can be set at config.properties
     private static final String TS_IPEX_ENABLE = "ipex_enable";
+    private static final String TS_IPEX_DTYPE = "ipex_dtype";
+    private static final String TS_IPEX_CHANNEL_LAST = "ipex_channel_last";
+    
+    private static final String TS_IPEX_TORCHSCRIPT = "ipex_torchscript";
+    private static final String TS_IPEX_INPUT_TENSOR_SHAPES = "ipex_input_tensor_shapes";
+    private static final String TS_IPEX_INPUT_TENSOR_DTYPE = "ipex_input_tensor_dtype";
+    private static final String TS_IPEX_QSCHEME = "ipex_qscheme";
+    // oneDNN Graph fusion config option that can be set at config.properties 
+    private static final String TS_ONEDNN_GRAPH_FUSION_ENABLE = "onednn_graph_fusion_enable";
+    // launcher config option that can be set at config.properties 
```

Comment:
are these parameters more related to each model? If yes, they should be set in each model's configuration.